### PR TITLE
set new content for default landing page on lms

### PR DESCRIPTION
**Pull request by grozdanowski**
New default LMS index page content.

Screenshot:
new-default-lms-index-page

### CODE REVIEW

File: openedx/core/djangoapps/appsembler/sites/utils.py [hunk 1]
```diff
@@ -498,707 +498,626 @@ def _get_initial_page_elements():
...
+                        },
+                        {
+                        "element-type":"content-block",
+                        "element-path":"page-builder/elements/_content-block.html",
+                        "options":{
+                            "content":{
+                            "en":"<p>We recommend you take a look at our <a title=\"https://appsembler.com/blog/how-to-author-first-open-edx-course-webinar-recording/\" href=\"https://appsembler.com/blog/how-to-author-first-open-edx-course-webinar-recording/\">How to Author Your First Open edX Course in 30 Minutes</a> blog post and the <a title=\"https://appsembler.wistia.com/medias/l5wyg0ua6l\" href=\"https://appsembler.wistia.com/medias/l5wyg0ua6l\">30 minute tutorial video from our Webinar</a>!</p>\n<p>For more, visit our <a title=\"http://help.appsembler.com/\" href=\"http://help.appsembler.com/\">Appsembler knowledge base</a>.</p>\n<p>If you run into any issues or have any questions about using our product, please feel free to contact our support team via <a title=\"mailto:support@appsembler.com\" href=\"mailto:support@appsembler.com\"><strong>support@appsembler.com</strong></a>!</p>"
```
Comment (OmarIthawi) [inline openedx/core/djangoapps/appsembler/sites/utils.py:1112]: We're still showing Open edX here. That's intentional, right?

**Review by OmarIthawi — APPROVED** [review]
I've fixed tests that this PR broke it. Please pull locally if you need to edit this branch.

**Comment by OmarIthawi** [comment]
Let's test it on staging. Code review isn't good for such PRs.

File: openedx/core/djangoapps/appsembler/sites/utils.py (hunk 1 above)
Comment (grozdanowski) [reply openedx/core/djangoapps/appsembler/sites/utils.py]: Yeah, it's the webinar title I guess.

**Pull request merged**